pkg/gce-pd-csi-driver/controller_test.go

```diff
@@ -988,7 +992,11 @@ func TestListVolumeArgs(t *testing.T) {
 			var d []*gce.CloudDisk
 			for i := 0; i < diskCount; i++ {
 				// Create 600 dummy disks
-				d = append(d, gce.CloudDiskFromV1(&compute.Disk{Name: fmt.Sprintf("%v", i)}))
+				name := fmt.Sprintf("disk-%v", i)
+				d = append(d, gce.CloudDiskFromV1(&compute.Disk{
+					Name:     name,
+					SelfLink: fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/project/zone/zone/disk/%s", name),
```

Comment:
Shouldn't it be projects/project/**zones**/zone ?

Reply:
ah, thanks. That's not causing the tests to fail though, since we don't validate that part of the resource (there are many options for that---regional, global, ...?